scripts/run_tests.sh

```diff
@@ -23,8 +23,14 @@ function display_result {
 make test-requirements
 display_result $? 1 "Requirements check"
 
+black ./app ./tests --check
```

Comment:
Changed to `black --config pyproject.toml --check .` after

Reply:
Nice!